components/cosmetic_filters/resources/data/BUILD.gn

```diff
@@ -1,12 +1,23 @@
+# Copyright (c) 2020 The Brave Authors. All rights reserved.
```

Comment:
It's a bit strange that these files are in `resources/data` since it's not data but is executed scripts / UI. How about `resources/ui`?